bin/README.md

```diff
@@ -0,0 +1,14 @@
+This folder contains scripts for performing various tasks related to the MARTI dice server.
+
+## Create and initialize the staging database
+
+1. Login to `tripleawarclub.org`.
+1. Change to the directory where you have cloned the `triplea-game/dice-server` repo.
```

Comment:
I'd appreciate if the list numbers would be counting up instead of staying at 1

Reply:
I believe this is standard markdown [syntax](https://guides.github.com/features/mastering-markdown/#syntax) for an ordered list.  A conforming markdown viewer should automatically increment the item number when rendered.  You can see how GitHub renders this file [on my branch](https://github.com/ssoloff/triplea-game-dice-server/tree/add-instructions-for-creating-staging-db/bin).